shotover/src/transforms/cassandra/sink_cluster/node_pool.rs

```diff
@@ -198,7 +199,7 @@ impl NodePool {
             })?,
             version,
         )
-        .unwrap();
+        .ok_or(GetReplicaErr::NoRoutingKey)?;
```

Comment:
CassandraSinkCluster keeps track of the version, lets make use of that version to log a `tracing::error` if NoRoutingKey occurs and we are on a version other than v3.
Otherwise we could start silently losing routing without ever realizing.

Reply:
Yeah, I'll make that change. This is unlikely though, we were unwrapping before and hit no issues.

Reply:
Right, i'm more concerned about accidentally breaking things in the future.
Say a new version of cassandra changes how this works or we change some of our routing implementation in shotover.